credit_card_checker/check.rb

```diff
@@ -3,7 +3,8 @@
 if ARGV.empty?
   p 'You should pass credit card number in params'
 else
+  ccc = CreditCardChecker.new
   ARGV.each do |code|
-    p [code, CreditCardChecker.valid?(code)]
+    p [code, ccc.valid?(code)]
```

Comment:
why don't you pass code to object's attribute?
credit_card_checker = CreditCardChecker.new(code)
credit_card_checker.valid?